bash.exe.stackdump

```diff
@@ -0,0 +1,9 @@
+Stack trace:
+Frame        Function    Args
+000FFFFA358  0018006021E (00180252DED, 001802340A6, 000FFFFA358, 000FFFF9250)
+000FFFFA358  00180048859 (00000000000, 00000000000, 00000000000, 001005D1D3C)
+000FFFFA358  00180048892 (00180252EA9, 000FFFFA208, 000FFFFA358, 00000000000)
+000FFFFA358  001800AF0D8 (00000000000, 00000000000, 00000000000, 00000000000)
+000FFFFA358  001800AF25D (000FFFFA370, 00000000000, 00000000000, 00000000000)
+000FFFFA5E0  001800B0673 (000FFFFA370, 00000000000, 00000000000, 00000000000)
+End of stack trace
```

Comment:
Is this an intended change? Please commit your files individually, and stash anything you do not intend to change. Delete this file on your end, and make a new commit please. That will automatically be added to this PR. Let me know if you need help.